tests/js/client/dump/dump-test.inc

```diff
@@ -1563,6 +1563,38 @@
         assertEqual(graph.orphanCollections.length, 0);
       },
 
+      testSmartGraph3_11_compat: function () {
+      return;
+        if (!isEnterprise) {
+          return;
+        }
+        const smartGraphName = "UnitTestDumpSmartGraphCompat";
+        const graphsColl = getCollection('_graphs');
+        assertTrue(!!graphsColl.exists(smartGraphName));
+
+        const graphDoc = graphsColl.document(smartGraphName);
+        // This is the requirement to test if the compat cid has effect
+        assertTrue(graphDoc.hasOwnProperty('initialCid'));
```

Comment:
IIRC, the enterprise part of the PR will remove this attribute. Is this assertion still valid now?
```suggestion
        assertTrue(graphDoc.hasOwnProperty('initialCid'));
```

Reply:
Yes in this case it absolutely is.
Because we are simulating an old graph here.
The corresponding setup could should inject the initialCid value like an old version would. (So after restore of old data, or rolling upgrade, we don't want to break either)
And this test should then validate that we do not have negative side effects by having this value set.

I updated the comment above